system/readline/readline_common.c

```diff
@@ -708,11 +708,6 @@ ssize_t readline_common(FAR struct rl_common_s *vtbl, FAR char *buf,
           buf[nch++] = '\n';
           buf[nch]   = '\0';
 
-#ifdef CONFIG_READLINE_ECHO
```

Comment:
should we remove CONFIG_READLINE_ECHO?

Reply:
Maybe should rename it to represent its function now (cmd history/backspace etc)

Reply:
understand, let's continue use CONFIG_READLINE_ECHO represent other echo.